lib/python/qmk/cli/cformat.py

```diff
@@ -26,7 +26,8 @@ def cformat(cli):
     else:
         for dir in ['drivers', 'quantum', 'tests', 'tmk_core']:
             for dirpath, dirnames, filenames in os.walk(dir):
-                if 'tmk_core/protocol/usb_hid' in dirpath:
+                ignores = ['tmk_core/protocol/usb_hid', 'quantum/template']
```

Comment:
```suggestion

```
I'd move this line out of the loops, right after `else` and would use Path objects:
`ignores = [Path('tmk_core/protocol/usb_hid'), Path('quantum/template')]`

Reply:
Moved.

Not sure I like the mixture of Path and strings, and would prefer to defer it to another round of #7872.

Reply:
Well, you can use `os.path`, something like `if os.path.join(dirpath, dn) in ignores`.

But it's just me thinking about the code, you original code worked enough, with 7b97f2d it works better, so I'm cool with it.